xml/decoders/Digitrax_DS74.xml

```diff
@@ -9,43 +9,62 @@
                 >
 
     <version author="Bob Jacobsen" version="1" lastUpdated="20230205" />
+    <version author="Bob Milhaupt" version="2" lastUpdated="20240708" />
 
     <decoder>
-
         <family name="Series 7 Boards" mfg="Digitrax">
             <model model="DS74"/>
         </family>
 
-        <programming direct="no" paged="yes" register="no" ops="no"><!-- paged yes allows initial setup and change of address -->
+        <programming direct="no" paged="no" register="no" ops="no">
+            <!-- paged yes allows initial setup and change of address -->
```

Comment:
Paged is changed from yes to no, but the comment is still there. Is this correct?

Reply:
Good catch!  the comment is correct and necessary.
Thanks!